gatsby-config.js

```diff
@@ -25,6 +25,7 @@ module.exports = {
       ETC: '0x1934945354BDe5b34F0c68E6AA78492050856D38',
     },
     facebookAppID: '2127644397453206',
+    facebookAccessToken: 'EAAID1NAqRPIBALqoVfwnWHFeIO4kpVXz8s95ZBG0u3NcCYNZBQdaDZCVDTp394tgJcTH8X7fWH5LvOJ9moHgtwzzmR78BGCJGZAM7loiDXITVNdezWfcfjAOeWW1vbtWTdEMBEM2xcRg48jI9RQFZCRrWONwEM6t8SYBk1Kc3wQZDZD',
```

Comment:
It is long-lived, but expires after ~90 days.
Todo: figure out how to automate refreshing this token automatically.

https://developers.facebook.com/docs/facebook-login/access-tokens/refreshing

Reply:
Is it safe to have it public? I thought it is supposed to be secret, like instagram token (stored in Netlify admin)

Reply:
No I don't want to have it public, I tried putting it into env variables and also prefixing it with `GATSBY_`, didn't make it work, but since for now I'm just testing things out and the fb app is not public, I just dumped it there. Will move it to some private place later.